HeadsAndTails/Program.cs

```diff
@@ -75,11 +75,14 @@ static void Main(string[] args)
                 Console.WriteLine(resultMessage);
 
                 Console.WriteLine();
-                Console.WriteLine("Нажмите Enter для начала новой игры...");
+                Console.WriteLine("Нажмите Enter для начала новой игры... Чтобы выйти нажмите Escape");
```

Comment:
Split the message text into two lines

Reply:
[https://github.com/Mallusca/heads-and-tails/pull/4/commits/79da79dc683960572de06329bbc54ac309a890e5](Fix)